src/common.ts

```
@@ -0,0 +1,261 @@
+  }
+
+  /**
+   * Move the offset and the line number down {@link n} times.
+   *
+   * This will also reset the column number to `0`.
+   *
+   * @param n The number of times to move the offset and line number forward.
+   * @returns The new offset.
+   * @throws Error if {@link n} is less than `0`.
+   */
+  public line(n: number = 1): number {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)